JASP-Engine/JASP/R/descriptives.R

```diff
@@ -977,14 +978,24 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 
 .plotMarginal <- function(column, variableName,
-                          rugs = FALSE, displayDensity = FALSE) {
+                          rugs = FALSE, displayDensity = FALSE, binWidthType) {
   column <- as.numeric(column)
   variable <- na.omit(column)
 
   if(length(variable) == 0)
     return(NULL)
+  
+  if(binWidthType == "doane"){
+    sigma.g1 <- sqrt((6*(length(variable) - 2)) / ((length(variable) + 1)*(length(variable) + 3)))
+    g1 <- mean(abs(variable)^3)
+    k <- 1 + log2(length(variable)) + log2(1 + (g1 / sigma.g1))
+    binWidthType <- k
+  }
+  
+  if(binWidthType == 'fd' & nclass.FD(variable) > 10000) # FD-method will produce extreme number of bins and crash ggplot, mention this in footnote
```

Comment:
```suggestion
  if (binWidthType == "fd" & nclass.FD(variable) > 10000) # FD-method will produce extreme number of bins and crash ggplot, mention this in footnote
```
either use `'` or `"` but don't mix them. Since the function uses `"` already, stick with `"`.